src/components/core/datetime/date-adapter.ts

```diff
@@ -145,7 +145,7 @@ export interface DateAdapter<T = any> {
   /** Get the given string as Date.
    * @param value The date in the format DD.MM.YYYY.
    */
```

Comment:
```suggestion
  /** Get the given string as Date.
   * @param value The date in the format DD.MM.YYYY.
   * @param now The current date as Date.
   */
```

Or maybe something like that

Reply:
fixed, thanks!